movies/src/main/java/com/fundacionjala/movies/Main.java

```diff
@@ -0,0 +1,14 @@
+package com.fundacionjala.movies;
+
+public class Main {
```

Comment:
Remove this unnecessary class and create a new unit test for the statement method